### ISSUE TITLE
Registration works but getting errors in log

### ISSUE THREAD

**Issue by Nestade** [pellaeon/registration#66]
Hi,

First of all the registration is working - I received the mail without any problems and was able to create an account. But I'm also getting some errors in my log (attached a screenshot).

![log](https://cloud.githubusercontent.com/assets/2645357/24086567/d54eb31c-0d11-11e7-8c4e-e98effbe7b27.JPG)

Am I missing any dependencies or are there any permissions wrong?
Any idea how I can fix it? 

```
Error	PHP	Undefined index: entered_data at /var/www/nextcloud/apps/registration/templates/form.php#24
Error	PHP	Undefined index: errormsgs at /var/www/nextcloud/apps/registration/templates/form.php#6
```
This errors appear after clicking the confirmation link

```
Error	PHP	fgets(): SSL operation failed with code 1. OpenSSL Error messages: error:0906D06C:PEM routines:PEM_read_bio:no start line error:0906D06C:PEM routines:PEM_read_bio:no start line error:0906D06C:PEM routines:PEM_read_bio:no start line error:0906D06C:PEM routines:PEM_read_bio:no start line error:0906D06C:PEM routines:PEM_read_bio:no start line error:0906D06C:PEM routines:PEM_read_bio:no start line error:0906D06C:PEM routines:PEM_read_bio:no start line at /var/www/nextcloud/3rdparty/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php#163
```

and this one appears after creating the account.

**OS:** Debian 8.7
**Webserver:** Apache2
**Database:** MySQL

**PHP 7.0 Modules:**
calendar
Core
ctype
curl
date
dom
exif
fileinfo
filter
ftp
gd
gettext
hash
iconv
igbinary
imagick
intl
json
libxml
mbstring
mcrypt
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
Phar
posix
readline
redis
Reflection
session
shmop
SimpleXML
sockets
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
wddx
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zip
zlib

Regards,
Nestade

**Comment by NextGenerationcloud** [comment]
Have the same -.-

**Comment by pellaeon** [comment]
Don't worry about the undefined index error, it won't affect anything. As
for the SSL error, it has nothing to do with registration plugin.

2017-04-19 11:14 GMT+08:00 NextGenerationcloud <notifications@github.com>:

**Comment by NextGenerationcloud** [comment]
Hello THX 

I had the error before not

Where does he comes from ?

**Comment by pellaeon** [comment]
This is fixed as of v0.3.0
